client/src/components/Card.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import colors from '../utils/colors';
+import PropTypes from 'prop-types';
+import Input from './Input';
+
+const CardContainer = styled.div`
+  width: 300px;
+  height: 430px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 2px solid ${colors.complementaryPrimary};
+  border-radius: 24px;
+  background-color: ${colors.backgroundPrimary};
+  font-size: 20px;
+  color: ${colors.fontPrimary};
+  box-shadow: 10px 15px 10px ${colors.backgroundPrimary};
+`;
+
+const CardTitleContainer = styled.div`
+  width: 274px;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+  padding: 10px;
+  font-size: 1.5rem;
+  border-bottom: 1px solid ${colors.complementaryPrimary};
+`;
+
+const CardDataContainer = styled.div`
+  width: 234px;
+`;
+
+const CardDataPriceContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 40px;
+`;
+
+const CardDataStockContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 40px;
+`;
+
+const CardDataValueContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 40px;
+`;
+
+const DataKeyValue = styled.div`
+  width: 140px;
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 15px;
+`;
+
+const StockInput = styled(Input)`
+  text-align: end;
+  border-radius: 14px 0 0 14px;
+  padding-right: 2px;
+`;
+
+const FlexContainer = styled.div`
+  display: flex;
+  max-width: 140px;
+  > input {
+    width: 140px;
+  }
+`;
+
+export default function Card({ title, priceValue, id, valueValue }) {
```

Comment:
I think you can leave the "Value" out, if the component name is more specific it might make more sense if the props are just `value` and `price`. For example if the `Card` would be named `CoinInfoCard` a prop called price is pretty much self explanatory.
I don't want to be annoying, but I think the value of a bitcoin is also its price, right? :P